pkg/render/logstorage/linseed/linseed.go

```diff
@@ -427,6 +432,9 @@ func (l *linseed) linseedDeployment() *appsv1.Deployment {
 	if l.cfg.ElasticClientSecret != nil {
 		annotations["hash.operator.tigera.io/elastic-client-secret"] = rmeta.SecretsAnnotationHash(l.cfg.ElasticClientSecret)
 	}
+	if l.cfg.ElasticClientCredentialsSecret != nil {
+		annotations["hash.operator.tigera.io/elastic-client-credential-secret"] = rmeta.SecretsAnnotationHash(l.cfg.ElasticClientCredentialsSecret)
```

Comment:
I suggest the name for this annotation also to be "tigera-ee-linseed-elasticsearch-user-secret", just like the name of the secret that it is using.